configs/template_config.yaml

```diff
@@ -59,35 +92,20 @@ activation_dropout: 0.0
 batch_size: 8
 # Checkpoints (bool): Whether to save checkpoints during training
 checkpoints: false
-# Device Configuration (str)
-device: "cuda" # Choices: "cpu", "cuda"
 # Dropout (float): Probability of an element to be zeroed during dropout
 dropout: 0.1
 # Embedding Dimension (int): Embedding dimension size of each token
 embed_dim: 80
-# Epochs (int): Number of epochs to train for
-epochs: 1
 # FFN Dimension (int): Hidden layer size of Feed Forward Network (FFN)
 ffn_dim: 12
-# Fully Sharded Data Parallel (bool): Whether to shard Module parameters across
-# data parallel workers or not (with the FairScale library)
-fsdp: false
 # Heads (int): Number of heads. Head architecture changes based on model
 heads: 4
 # Layers (int): Number of stacked network layers
 layers: 2
-# Learning Rate (float)
-learning_rate: 0.001
 # Model Type (str): Name of model architecture to train
-model_type: "retnet" # Choices: "retnet", "transformer"
-# Number of GPUs (int)
-num_devices: 1
-# Random Seed (int): For reproducibility
-rand_seed: 42
+model_type: retnet # Choices: "retnet", "transformer"
```

Comment:
Is there a reason why we're making strings no longer in double quotes (ex: `"retnet"`)?

Reply:
Technically quotes are unnecessary in yaml, but changed for consistency